src/commands/utility/hex.ts

```diff
@@ -24,37 +25,28 @@ export default class extends Command {
                 ? Math.floor(Math.random() * 16777215).toString(16)
                 : color;
 
-        const buffer = new Canvas(200, 100)
-            .setColor(`#${hex}`)
-            .addRect(0, 0, 200, 100)
-            .setColor(this.bw(hex))
-            .setTextFont('20px Impact')
-            .setTextAlign('left')
-            .addText(`#${hex}`.toUpperCase(), 5, 95)
-            .toBuffer();
+        const json = await fetch(`https://canvas.typicalbot.com/api/v1/color?hex=${hex}`).then((body) => body.json());
+
+        fs.mkdir('data', (err) => {
+            if (err && err.code !== 'EEXIST') console.error(err);
+        });
```

Comment:
I feel like this should be elsewhere instead of here in case we use the `data` directory for something else.